crmsh/schema.py

```diff
@@ -153,4 +153,12 @@ def rng_xpath(xpath, namespaces=None):
     return _crm_schema.rng_xpath(xpath, namespaces=namespaces)
 
 
+def compare_with(schema1, schema2):
```

Comment:
It looks like I'm paranoid on the naming today, and I think the better name here is 
```suggestion
def compare(schema1, schema2):
```

In theory, in the different context though no necessary here, it could be 
```
def compare_with(schema1):
```

Reply:
Changed